### ISSUE TITLE
add vessel name to flight comp title

### ISSUE THREAD

**Issue by krenshala** [RemoteTechnologiesGroup/RemoteTech#673]
This is purely a feature request.

Due to the way the flight computer window behaves - continuing to issue commands to the active vessel when it was opened, and not necessarily the currently active vessel - I think it would be helpful to have the Flight Computer window title reflect the craft it will be issuing commands to.

Changing the title from the current text, "FlightComputer", to something like "FlightComputer: <vessel-name>" would prevent confusion when switching vessels, as it would show players they need to close and reopen the window to issue commands to the currently selected vessel.

Alternately, update the code so the flight computer window always issues commands to the currently active vessel, and not the vessel that was active when that window was opened, assuming it meets all the normal requirements for using the flight computer (has comms and power, etc).

**Comment by CosineLoss** [comment]
Seconded.

I'd prefer the first option, with a huge red blinking warning (slight exaggeration...) warning me that I'm about to mess with a slaved FC.

**Comment by petersohn** [comment]
I would prefer automatically closing the flight computer window when I leave the scene. First, because the window's close button is disabled when I'm not at an active flight (e.g. at KSC). Second, because it happened to me several times that I reloaded a quicksave while the flight computer window was open, then forgot that that window does not correspond to my current vessel. I kept issuing commands and failed to see why it doesn't work. Displaying the vessel name doesn't help here.
